src/index.html

```diff
@@ -1,23 +1,12 @@
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-
-        <link rel="stylesheet" href="css/style.css" />
-        <link
```

Comment:
Do we not need font-awesome anymore?

Reply:
Yes, we used it in the `pen, books, users ` and `social media` icons